### PR TITLE
feat(notifications): add missed call alert and friend request alert

### PR DESCRIPTION
**Pull request by Jekrimo**
**What this PR does** 📖
adds missed call alert, adds friend request alert

**Which issue(s) this PR fixes** 🔨

AP-1807 & AP-1532

### CODE REVIEW

**Comment by stavares843** [comment]
good point 🔨 i'd say if the user declined the call, maybe we don't need the alert regarding a missing call

**Comment by Jekrimo** [comment]
> good point 🔨 i'd say if the user declined the call, maybe we don't need the alert regarding a missing call

Good call! Got a fix up!

**Review by Clayton215 — APPROVED** [review]

File: store/textile/actions.ts [hunk 1]
```diff
@@ -212,91 +202,13 @@ export default {
     // TODO: only for testing
     dispatch('subscribeToMailbox')
   },
-  addMessageToConversation(
```
Comment (molimauro) [inline store/textile/actions.ts:215]: Why this has been deleted?

File: store/textile/actions.ts [hunk 2]
```diff
@@ -326,8 +233,7 @@ export default {
       if (!sender) {
         return
       }
-
-      dispatch('addMessageToConversation', {
```
Comment (molimauro) [inline store/textile/actions.ts:330]: also, why these changes?

**Review by molimauro — CHANGES_REQUESTED** [review]

**Comment by molimauro** [comment]
The notifications part looks neat!! Some questions about the code not related to this PR ;)

**Comment by molimauro** [comment]
Mmmm @stavares843 why are we merging dev in the PRs?

@Jekrimo if they are not related to this PR id try to remove them, can you try it? Or close this one and keep only the commit related to your work.. Thanks!

**Comment by stavares843** [comment]
because rebase wasn't available due to conflicts @molimauro 

merging dev didn't deleted anything, it just updated the PR with the latest dev


as you can see on dev branch - screenshot below, these lines are available on dev

Captura de ecrã 2022-07-05, às 15 16 58



but they are being removed on this PR, so these changes are related to the PR, and not from merging dev or rebasing or anything similar



Captura de ecrã 2022-07-05, às 15 15 48

**Comment by stavares843** [comment]
such as if you go to the initial commits, that was already removed here on the first commit

https://github.com/Satellite-im/Core-PWA/pull/3806/commits/f43b461fc393102f08deac6706b6897a137808a8#diff-263e7ed642b6b3ea140ee1955884bf4d42f4cfc4f286eac45d08f7bac78dcf7dL203

**Comment by josephmcg** [comment]
There are definitely issues here. If it can't be rebased without solving conflicts, a dev should rebase locally. A lot of files work i did recently is being removed

**Comment by josephmcg** [comment]
It seems like conflicts were 'resolved' by accepting current changes and disregarding incoming, rather than picking what we need from both

**Comment by stavares843** [comment]
@josephmcg 

if you go to the initial commits, that was already removed here on the first commit

https://github.com/Satellite-im/Core-PWA/commit/f43b461fc393102f08deac6706b6897a137808a8#diff-263e7ed642b6b3ea140ee1955884bf4d42f4cfc4f286eac45d08f7bac78dcf7dL203


its not related to any rebase or merging dev 😅

**Comment by Jekrimo** [comment]
I think that fixed it?

**Comment by molimauro** [comment]
@Jekrimo this one is needed? `store/textile/actions.ts`

File: store/textile/actions.ts [hunk 3]
```diff
@@ -156,6 +156,15 @@ export default {
       ]
     }
 
+    // Since this is async, we have to check that we are still in the chat, if
```
Comment (stavares843) [inline store/textile/actions.ts:159]: can we add the comments using multi line? 💥 

/**

**Comment by WanderingHogan** [comment]
closing, but leaving branch in case we come back to it when we integrate notifications in iridium

**Pull request closed without merging**